src/main/java/uk/ac/ebi/ega/egacryptor/pipeline/DefaultCryptographyPipeline.java

```diff
@@ -90,13 +91,13 @@ private void doProcess(final FileToProcess fileToProcess) throws CryptographyExc
         final MessageDigest inputStreamMessageDigest = Hash.getMD5();
         final MessageDigest outputStreamMessageDigest = Hash.getMD5();
 
-        final DigestInputStream digestInputStream = new DigestInputStream(new FileInputStream(inputFile), inputStreamMessageDigest);//Will be closed in PipelineStream
         long bytesRead;
 
-        try (final DigestOutputStream digestOutputStream = new DigestOutputStream(new FileOutputStream(outputFileGPG), outputStreamMessageDigest)) {
-            final OutputStream pgpEncryptedOutputStream = cryptography.encrypt(digestOutputStream);//Will be closed in PipelineStream
+        try (final DigestInputStream digestInputStream = new DigestInputStream(Files.newInputStream(inputFile.toPath()), inputStreamMessageDigest);
+             final DigestOutputStream digestOutputStream = new DigestOutputStream(Files.newOutputStream(outputFileGPG.toPath()), outputStreamMessageDigest)) {
+            final OutputStream pgpEncryptedOutputStream = cryptography.encrypt(digestOutputStream);
             try (final PipelineStream pipelineStream = new DefaultStream(digestInputStream, pgpEncryptedOutputStream, bufferSize)) {
-                LOGGER.info("File {} is being processed", inputFile.getPath());
+                LOGGER.info("File \"{}\" is being processed", inputFile.getPath());
```

Comment:
Some tests passed an empty dir here, which made the log message look like this: `File  is being processed`. Now it looks like this: `File "" is being processed`.